lib/elixir/test/erlang/module_test.erl

```diff
@@ -100,6 +100,11 @@ nested_ref_test() ->
 module_with_elixir_as_a_name_test() ->
   ?assertError(#{'__struct__' := 'Elixir.CompileError'}, eval("defmodule Elixir do\nend")).
 
+module_with_special_atom_as_a_name_test() ->
```

Comment:
We rarely add new tests to the Erlang suite. Can you please add those to lib/elixir/test/elixir/kernel_test.exs instead? Thanks!

Reply:
Moved it to [kernel_test.exs](https://github.com/elixir-lang/elixir/pull/9032/files#diff-8899a6e958f0940b9f6b6e83ca6184ddR630)